packages/state/src/recs/index.ts

```
@@ -159,37 +159,37 @@
+            no_hashed_keys: false,
+            models: entityModels,
+            historical,
+        });
 
         if (dbConnection) {
-            await insertEntitiesInDB(dbConnection, entities);
+            await insertEntitiesInDB(dbConnection, entities.items);
         }
 
-        if (logging) console.log(`Fetched entities`, entities);
+        if (logging) console.log(`Fetched entities`, entities.items);
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)